packages/flutter/lib/src/gestures/drag.dart

```diff
@@ -63,10 +63,22 @@ bool _isFlingGesture(Velocity velocity) {
 }
 
 abstract class _DragGestureRecognizer<T extends dynamic> extends OneSequenceGestureRecognizer {
```

Comment:
(while you're here, I'm pretty sure "T extends dynamic" is the same as just saying "T" here.)